app/src/main/java/com/amaze/filemanager/adapters/data/LayoutElementParcelable.java

```diff
@@ -141,10 +141,10 @@ public LayoutElementParcelable(Parcel im) {
         symlink = im.readString();
         int j = im.readInt();
         date = im.readLong();
+        dateModification = im.readString();
```

Comment:
This line is in the wrong position, I think, the `Parcel` must be read in the same order it is written.